tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -42,21 +43,32 @@
 # TODO: Update baseline values after fix.
 CREATE_LATENCY_BASELINES = {
     "x86_64": {
-        "2vcpu_256mb.json": {"FULL": {"target": 180}, "DIFF": {"target": 70}},
-        "2vcpu_512mb.json": {
-            "FULL": {"target": 280},
-            "DIFF": {"target": 75},
+        "m5d.metal": {
+            "2vcpu_256mb.json": {"FULL": {"target": 180}, "DIFF": {"target": 70}},
+            "2vcpu_512mb.json": {
+                "FULL": {"target": 280},
+                "DIFF": {"target": 75},
+            },
+        },
```

Comment:
Aren't values for m6i.metal also needed?
```suggestion
        },
        {
            "2vcpu_256mb.json": {"FULL": {"target": 180}, "DIFF": {"target": 70}},
            "2vcpu_512mb.json": {
                "FULL": {"target": 280},
                "DIFF": {"target": 75},
            },
        },
```

Reply:
I added a TODO to also gather actual numbers for m6i.
I made this change so that I am able to modify changes per processor type.
It is not in the scope of this PR to update m6i values.